he7d3r/kedro-website
# Remove duplicated file from project template

## Description

The project template example shows this line twice:
```
├── setup.cfg       # Configuration options for testing and linting
```

## Development notes

Removed one of the copies from the example.

## QA notes

Compare the results of the URLs before and after the change. There should be only one `setup.cfg`.

## Checklist
